Documentation/requirements.txt

```diff
@@ -19,6 +19,8 @@ six==1.15.0
 snowballstemmer==1.2.1
 Sphinx==1.8.1
 sphinx-autobuild==0.7.1
+# forked read the docs themez
+git+git://github.com/cilium/sphinx_rtd_theme.git
```

Comment:
I guess this pulls the master branch from that repo, might be good to pin it to a specific version instead. As-is, we'll end up caching a random version inside the docs-builder image and it's a bit counter-intuitive to update the docs-builder since I think that docker won't detect any difference by default (since we're not changing any files that the Dockerfile references).

Reply:
Updated, now it points to the specific commit